ios/RNTweetShadowView.m

```diff
@@ -4,23 +4,24 @@ @implementation RNTweetShadowView
 
 - (TWTRTweetView *)tweetViewForMeasuring
 {
-    static TWTRTweetView *tweetView = nil;;
+    static TWTRTweetView *tweetView = nil;
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
         tweetView = [[TWTRTweetView alloc] initWithTweet:nil style:TWTRTweetViewStyleCompact];
```

Comment:
The problem also arises here, `dispatch_sync`  should be called instead of `dispatch_once`.

Reply:
Well, alloc can be safely called in a bkgnd thread. The initWithTweet is bkgnd thread is safe also – at least thats what Apple’s code analyser says.